server/db/connection.js

```diff
@@ -1,18 +1,34 @@
+// import mongoose from "mongoose";
+
+// // Function to establish connection with MongoDB
```

Comment:
Seems like this is the old commented code. I think it is better to use Git/version control to keep track of previous versions of code. Can you delete this commented code?